pyro/contrib/gp/kernels/brownian.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, input_dim, variance=None, active_dims=None, name="Brownian"):
         super(Brownian, self).__init__(input_dim, active_dims, name)
 
         if variance is None:
-            variance = torch.ones(1)
+            variance = torch.tensor(1)
```

Comment:
We will need to make this a float tensor here and elsewhere `torch.tensor(1.)`. This is necessary after https://github.com/pytorch/pytorch/pull/5997. See #944 for our migration plans.